From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brian Behlendorf <behlendorf1@llnl.gov>
Date: Thu, 23 Aug 2018 09:34:34 -0700
Subject: [PATCH] Add libaio-devel BuildRequires

The zfs-test package needs a build requirement on the libaio-devel
package.  Without it ./configure will correctly determine that
mmap_libaio cannot be built and it will be skipped.

Reviewed-by: George Melikov <mail@gmelikov.ru>
Signed-off-by: Brian Behlendorf <behlendorf1@llnl.gov>
Closes #7821
Closes #7824

Signed-off-by: Stoiko Ivanov <s.ivanov@proxmox.com>
---
 rpm/generic/zfs.spec.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/generic/zfs.spec.in b/rpm/generic/zfs.spec.in
index 398221c6..16c5780b 100644
--- a/rpm/generic/zfs.spec.in
+++ b/rpm/generic/zfs.spec.in
@@ -197,6 +197,7 @@ Requires:       sudo
 Requires:       sysstat
 Requires:       rng-tools
 Requires:       libaio
+BuildRequires:  libaio-devel
 AutoReqProv:    no
 
 %description test
